tests/5.0/target_update/test_target_update_mapper_from_discontiguous.c

```diff
@@ -0,0 +1,56 @@
+//===--- test_target_update_mapper_from_discontiguous.c -----------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// This test seeks to ensure that target update with motion-clause "from" can properly
+// map data from the device by specifying a user-defined mapper. Additionally, the test
+// checks a new addition to target update in OpenMP 5.0 that states "List items in the
+// to or from clauses may include array sections with stride expressions." 
```

Comment:
Where is the strided expression used ?
The format should be: 
[ lower-bound : length : stride] or
[ lower-bound : : stride] or
[ : length : stride] or
[ : : stride]